diff -ur a/setup.py b/setup.py
--- a/setup.py  2022-10-05 12:26:36.341842132 +0000
+++ b/setup.py  2022-10-05 12:28:24.807379111 +0000
@@ -22,13 +22,11 @@
         'docutils>=0.14',
         'python-daemon>=2.3.0',
         'wheel>=0.32.3',
-        'backports.shutil_which>=3.5.1',
         'ConfigArgParse>=0.12.1',
         'python-daemon>=2.1.2',
         'ecdsa>=0.13',
         'pynacl>=1.4.0',
         'mnemonic>=0.18',
-        'pymsgbox>=1.0.6',
         'semver>=2.2',
         'unidecode>=0.4.20',
     ],
