diff -urN linux-gui-2.1.1-orig/setup.py linux-gui-2.1.1/setup.py
--- linux-gui-2.1.1-orig/setup.py	2020-09-10 16:36:01.263346795 +0000
+++ linux-gui-2.1.1/setup.py	2020-09-10 16:36:31.772801789 +0000
@@ -50,7 +50,7 @@
     url="https://github.com/ProtonVPN/linux-gui",
     install_requires=[
         "protonvpn-cli>=2.2.2",
-        "requests>=2.23.0",
+        "requests",
     ],
     python_requires=">=3.5",
     classifiers=[
